main/ejml-simple/src/org/ejml/simple/SimpleMatrix.java

```diff
@@ -97,15 +97,6 @@ public class SimpleMatrix extends SimpleBase<SimpleMatrix> {
      * column-major.
      * </p>
      *
```

Comment:
Maybe just change it to vararg? I'm honestly not sure why it isn't either...

Reply:
OK, I've changed the constructors to accepts varargs and readded the explanation in the docstring.

Reply:
can you squash these changes? I'll then approve it.

Reply:
I've squashed the docstring changes, but kept the varargs change separate as it's the only change that affects the code. I can squash that too, if you prefer.